allennlp/nn/initializers.py

```diff
@@ -114,7 +114,8 @@ def from_params(cls, params: Params):
         "kaiming_uniform": _initializer_wrapper(torch.nn.init.kaiming_uniform),
         "sparse": _initializer_wrapper(torch.nn.init.sparse),
         "eye": _initializer_wrapper(torch.nn.init.eye),
-        "block_orthogonal": _initializer_wrapper(block_orthogonal)
+        "block_orthogonal": _initializer_wrapper(block_orthogonal),
+        "none": _initializer_wrapper(lambda x: x)
```

Comment:
What's the use case for this? Any parameter not matching a regex will use whatever it is initialised to in the code and this replicates that, so it seems a little redundant.